backend/LexBoxApi/GraphQL/LexQueries.cs

```diff
@@ -45,6 +54,14 @@ public async Task<IQueryable<Project>> ProjectByCode(LexBoxDbContext context, IP
         return context.Projects.Where(p => p.Code == code);
     }
 
+    [UseSingleOrDefault]
+    [UseProjection]
```

Comment:
This instead of `permissionService.AssertIsAdmin();`?
```suggestion
    [UseProjection]
    [AdminRequired]
```

Reply:
Sure, will do.

Reply:
Done in commit 02c8c4617dc0c16d03da777360870ddc2e57d0bc.